### DISCUSSION TITLE
ppocr svrt识别训练，用最新v6tiny训练或者en_PP-OCRv4_mobile_rec.yml训练，精度上不去，只有0.3

### DISCUSSION THREAD

**Discussion by zhangshabao** [PaddlePaddle/PaddleOCR#18255] [Q&A]
ppocr svrt识别训练，用最新v6tiny训练或者en_PP-OCRv4_mobile_rec.yml训练，精度上不去，只有0.3
训练的样本6w张是模拟生成的数据，数据是长文本数据。现在精度上不去，
en_PP-OCRv4_mobile_rec.yml的配置
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.0001
    warmup_epoch: 2    
  regularizer:
    name: L2
    factor: 3.0e-05
Architecture:
  model_type: rec
  algorithm: SVTR_LCNet
  Transform: null
  Backbone:
    name: PPLCNetV3
    scale: 0.5
  Neck:
    name: SequenceEncoder
    encoder_type: reshape
  Head:
    name: CTCHead
    mid_channels: 64
    fc_decay: 0.00002
    return_feats: false
Loss:
  name: CTCLoss
  use_focal_loss: false
PostProcess:
  name: CTCLabelDecode
Metric:
  name: RecMetric
  main_indicator: acc
  ignore_space: false

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.0001
    warmup_epoch: 5
  regularizer:
    name: L2
    factor: 3.0e-05

PP-OCRv6/PP-OCRv6_tiny_rec.yml配置改成这样了
Architecture:
  model_type: rec
  algorithm: SVTR_LCNet
  Transform:
  Backbone:
    name: PPLCNetV4
    model_size: tiny
  # Head:
  #   name: CTCOnlyMultiHead
  #   head_list:
  #     - CTCHead:
  #         Neck:
  #           name: reshape
  #         Head:
  #           mid_channels: 80
  #           use_guide: true
  #           fc_decay: 0.00001
  Neck:
    name: SequenceEncoder
    encoder_type: reshape
  Head:
    name: CTCHead
    mid_channels: 96
    fc_decay: 0.00002
    return_feats: false

Loss:
  name: CTCLoss

PostProcess:
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc
帮忙看下哪里出了问题
